projects/aca-shared/src/lib/services/app.service.ts

```diff
@@ -177,6 +179,11 @@ export class AppService implements OnDestroy {
     });
 
     this.overlayContainer.getContainerElement().setAttribute('role', 'region');
+
+    const isMobileSwitchEnabled: boolean = this.config.get<string>('mobileAppSwitch.enabled', 'false').toLowerCase() === 'true';
```

Comment:
we have applied this approach for safety reasons if someone forgets to give value to enabled while configuration then also it will work as it is string, if u say we can also do with that approach.